webpack/components/withDeleteModal.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { translate as __, sprintf } from 'foremanReact/common/I18n';
+import ConfirmModal from './ConfirmModal';
+
+const withDelete = Component => {
```

Comment:
would this also be solved by theforeman/foreman#8619?

Reply:
I hope so.